app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -167,11 +188,29 @@ public void onEvent(Event event) {
                             new String[]{updatedFile.getPath()});
                 }
                 break;
+            case "ItemStarted":
+                break;
+            case "LocalIndexUpdated":
+                // ToDo
+                Log.v(TAG, "Unhandled event " + event.type);
+                break;
+            case "LoginAttempt":
+                break;
             case "Ping":
                 // Ignored.
                 break;
-            default:
+            case "RemoteIndexUpdated":
+                // ToDo
```

Comment:
Actually, having access to a computer and looking at the web UI code, it seems only LocalIndexUpdated triggers stuff, and none of the stuff it checks is visible in android (last scan, last item etc), so there is nothing todo under neither IndexUpdated events.

My bad, sorry.

Reply:
No problem, your info is still valuable for me working on future PR :)